src/Module/Register.php

```diff
@@ -333,6 +333,10 @@ protected function post(array $request = [])
 
 				if ($res) {
 					DI::sysmsg()->addInfo(DI::l10n()->t('Registration successful. Please check your email for further instructions.'));
+					$this->sendNotification($user, 'SYSTEM_REGISTER_NEW');
+					if (DI::config()->get('system', 'register_notification')) {
+						$this->sendNotification($user, 'SYSTEM_REGISTER_NEW');
+					}
```

Comment:
I believe there a copy/paste fail here.
```suggestion
					if (DI::config()->get('system', 'register_notification')) {
						$this->sendNotification($user, 'SYSTEM_REGISTER_NEW');
					}
```

Reply:
Yeah, slightly wrong :-)